samples/Grafana/README.md

```diff
@@ -0,0 +1,55 @@
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Flocalmachine)
+
+# Configure `dotnet monitor` in an AKS cluster for Managed Grafana
+
+`dotnet monitor` provides snapshots of .NET metrics in the Prometheus exposition format. [Prometheus](https://prometheus.io/docs/introduction/overview/) in turn collects metrics from targets by scraping metrics HTTP endpoints.
+
+This doc provides instructions on customizing metrics scraping for a Kubernetes cluster with the metrics addon in Azure Monitor and how to subsequently renders those metrics with Grafana.
+
+### Step 1: `dotnet monitor` configuration
+
+The following settings ensure that the [metrics endpoint is bound to an external address](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration/metrics-configuration.md#metrics-urls), and not the internal localhost. Today we also recommend setting a fairly low scraping interval for the metric count as follows.
+
```

Comment:
You can link to the github issue here (https://github.com/dotnet/dotnet-monitor/issues/4469)